OWNERS

```diff
@@ -0,0 +1,23 @@
+# See the OWNERS docs: https://go.k8s.io/owners
+
+approvers:
+- elsony
+- johnmcollier
+- kim-tsao
+- maysunfaisal
+- michael-valdron
+- thepetk
+- yangcao77
+- feloy
+- rm3l
```

Comment:
Question is if we need all these ppl on the approvers and review list?

Reply:
The idea was to add all the devfiles team inside the document

Reply:
The main idea is to add all the team devfile members to the list of OWNERS file so everyone is included :)

Reply:
But realistically the reason why I introduced this topic was to reduce spam review notifications..

Like I understand we want everyone on the team to contribute in some capacity. But if everyone is on the list, and the CI tags X and Y (2 people) for reviews, instead of @thepetk who is a frequent contributor, then we are merging code without the opinion of SME.

That was my thought process behind this.

Reply:
I've updated the list of approvers and reviewers :)